============================= test session starts =============================
platform win32 -- Python 3.10.11, pytest-8.3.4, pluggy-1.5.0 -- C:\Users\Sam\AppData\Local\Programs\Python\Python310\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\Sam\Desktop\Project-MORPH
configfile: pyproject.toml
testpaths: tests
plugins: xdist-3.6.1
collecting ... collected 34 items / 1 error

=================================== ERRORS ====================================
______________ ERROR collecting tests/test_continual_learning.py ______________
ImportError while importing test module 'C:\Users\Sam\Desktop\Project-MORPH\tests\test_continual_learning.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
..\..\AppData\Local\Programs\Python\Python310\lib\importlib\__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests\test_continual_learning.py:7: in <module>
    from morph.utils.data import ContinualTaskDataset
morph\utils\data.py:4: in <module>
    from torchvision import datasets, transforms
E   ModuleNotFoundError: No module named 'torchvision'
=========================== short test summary info ===========================
ERROR tests/test_continual_learning.py
!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
============================== 1 error in 1.46s ===============================
